fplll/defs.h

```diff
@@ -179,6 +190,8 @@ enum LLLMethod
 
 const char *const LLL_METHOD_STR[6] = {"wrapper", "proved", "heuristic", "fast"};
 
+const char *const HLLL_METHOD_STR[4] = {"wrapper", "proved", "", "fast"};
+
```

Comment:
What does the empty string signify?

Reply:
Since I keep the same `LLLMethod` for both LLL and HLLL, the method `LM_HEURISTIC` must be defined, even if it can not be called by the fplll binary, since a call to `fplll -a hlll -m heuristic` will automatically reach the check at the beginning of `hlll_reduction_z`. I can add at least a comment in `defs.h` to explain why the string is empty, but I can also set the empty string to `undefined`, for example.

Reply:
Hi, either of those options would be fine in my book.

Reply:
Modification provided in 8d30ee0328747f8bd2102d965b5d4848de344f74.